R/database.R

```diff
@@ -95,6 +104,9 @@ get_ssm_by_samples = function(these_sample_ids,
                               flavour = "clustered",
                               these_genes,
                               min_read_support = 3,
+                              basic_columns = TRUE,
+                              maf_cols = NULL,
+                              return_cols = FALSE,
```

Comment:
I understand the convenience option here but I wonder if this is better placed in a separate function. It could even be a global variable "maf_cols" if it was consistent or could be set as a global variable the first time this function or some other MAF-loading function is called.

Reply:
I'm envisioning a way for the user could set a global variable that is a list of their preferred columns by subsetting from the full global variable. If that was ensured to exist then the function could make use of it directly from their environment without even asking for it.

Reply:
I think it's fine for now as long as it works as advertised :)